tests/i18n/tests.py

```diff
@@ -1161,6 +1161,24 @@ def test_localize_templatetag_and_filter(self):
                 self.assertEqual(template2.render(context), output2)
                 self.assertEqual(template3.render(context), output3)
 
+    def test_localized_off_numbers(self):
+        """
+        Whatever the settings, unlocalized numbers should always be pure Python
+        representations.
+        """
+        template = Template(
+            '{% load l10n %}{% localize off %}{{ int }}/{{ float }}/{{ decimal }}{% endlocalize %}'
+        )
+        context = Context({'int': 1455, 'float': 3.14, 'decimal': decimal.Decimal('24.1567')})
+        with self.settings(
+            USE_L10N=True,
```

Comment:
Ticket describes an issue when `USE_L10N` is off, so I think we should check both combination:
```python
with self.settings(USE_L10N=True, DECIMAL_SEPARATOR=','):
    ...
with self.settings(USE_L10N=False, DECIMAL_SEPARATOR=','):
    ...
```
Please move tests for a thousand separator to a separate commit because they work without this patch.

Reply:
I will split them. At first glance I made a false assumption that all five affect `{% localize off %}` tag.